solutions/cf-1514C.mdx

```diff
@@ -0,0 +1,84 @@
+---
+id: cf-1514C
+source: CF
+title: Product 1 Modulo N
+author: Jesse Choe
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/89810)
+
+
+**Time Complexity:** $\mathcal{O}(N\log N)$
+
+<LanguageSection>
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+using ll = long long;
+
+ll gcd(ll a, ll b){
+	if(b == 0) return a;
+	return gcd(b, a % b);
+}
+
+int main(){
+	int n; cin >> n;
+	vector<ll> coprimes; 
+	/*
+	 * This array stores all of the numbers 
+	 * which are less than n and are coprime with n.
+	 */
+	for (int i = 1; i < n; i++) {
+		/*
+		 * A number i is coprime with n if gcd(i, n) = 1
```

Comment:
why use a multiline comment for a single line comment

Reply:
For consistency ig idk